hokusai/development.yml

```diff
@@ -0,0 +1,40 @@
+---
+version: '2'
+services:
+  force:
+    command: yarn start
+    environment:
+    - NODE_ENV=development
```

Comment:
I in general prefer that all config is stored in environment variables set at runtime, and am in favor of ditching any .env files that are under source control / need to live in 1Password or somewhere else when they contain secrets.